packages/ag-charts-enterprise/src/features/crosshair/crosshair.ts

```diff
@@ -1,10 +1,8 @@
-import { _ModuleSupport, _Scene, _Util } from 'ag-charts-community';
+import { type AgCrosshairLabelRendererResult, _ModuleSupport, _Scene, _Util } from 'ag-charts-community';
```

Comment:
We already import this elsewhere in this module, so seemed reasonable to use it here instead of the "shadowed" `any` definition.